templates/docker-daemon.json.j2

```diff
@@ -0,0 +1 @@
+{{ docker_config_options | default([])| to_nice_json }}
```

Comment:
Just thinking through this, let's call the variable `docker_daemon_json_options` instead of `docker_config_options`—it would be a little more directly related to the file under management.